pkgs/applications/networking/instant-messengers/gomuks/hardcoded_path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/notification/notify_linux.go b/lib/notification/notify_linux.go
+index f93a95f..ddc1a35 100644
+--- a/lib/notification/notify_linux.go
++++ b/lib/notification/notify_linux.go
+@@ -32,7 +32,7 @@ func Send(title, text string, critical, sound bool) error {
+ 		if critical {
+ 			soundName = "complete"
+ 		}
+-		exec.Command("paplay", "/usr/share/sounds/freedesktop/stereo/"+soundName+".oga").Run()
++		exec.Command("paplay", "/run/current-system/sw/share/sounds/freedesktop/stereo/"+soundName+".oga").Run()
```

Comment:
I wonder if we could use substituteAll in postPatch and reference `pkgs.sound-theme-freedesktop` for a proper not-hardcoded path.

Reply:
Sure. I'm not too familiar with `substituteAll`. It's the `@...@` thing right?

Reply:
Yep, you add a string to the replaced to the patch, and substitute a value from the patch phase.

```
  patches = [
    (substituteAll {
      src = ./yourpatchgoeshere.patch;
      replaceme = "${pkgs.sound-theme-freedesktop}/path/to/your/files";
    })
  ];
```

Reply:
Done. It seems like `substituteAll` doesn't support dashes in the attribute names...